tests/e2e.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(clippy::too_many_arguments)]
```

Comment:
Sanity-checking this

Reply:
The `update_profile` has more than 7 params, the ideal thing would be to create a struct, but because this is only for the E2E tests I don't think it matters much :p